### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by billymohajeri**
Hi 🙋‍♀️,
In this PR, I used a relational database to create the initial data structure for a vet clinic.

- Used the [template](https://github.com/microverseinc/curriculum-template-databases) to generate the repository.
- Created a database named vet_clinic.
- Created a table named animals with the columns (id, name, date_of_birth, escape_attempts, neutered, weight_kg).
- Inserted the given data into the table.
- Wrote queries for the given quests.
- Copied the SQL used to create the table into the schema.sql file in the generated repository.
- Copied the SQL used to insert data to the table into the data.sql file in the generated repository.
- Copied the SQL used to perform queries into the queries.sql file in the generated repository.
- Took screenshots from the results of my queries:

queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(250),
+  date_of_birt DATE,
```
Comment (rloterh) [inline schema.sql]: - [ ] Insert query is failing here due to mismatch of column spelling. Let's correct this by changing `date_of_birt` to `date_of_birth`,  as seen in the animal table.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-1-1' AND '2019-1-1';
```
Comment (rloterh) [inline queries.sql]: - [ ] Querying all animals born between 2016 and 2019, should be similar to:
```sql
SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-31-12';
```
Let's improve upon our query to match the above.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Billy, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  🏃 

## Highlights 👍  
💠 Good job so far implementing database table queries!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @billymohajeri 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**